### PR TITLE
Added ruleset for bibb.de

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

File: src/chrome/content/rules/BIBB.de.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	<target host="naa309.bibb.de" />
+	<target host="preview.nl.bibb.de" />
+	<target host="opac.bibb.de" />
+	<target host="prueferportal.bibb.de" />
+	<target host="stats.bibb.de" />
+	<target host="umfrage.bibb.de" />
+	<target host="wbmonitor.bibb.de" />
+	<target host="workspace.bibb.de" />
+	<target host="www2.bibb.de" />
+
```
Comment (YegorIevlev) [inline src/chrome/content/rules/BIBB.de.xml]:
```xml
<securecookie host=".+" name=".+" />
```

File: src/chrome/content/rules/BIBB.de.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		europass.bibb.de
+		exanda.bibb.de
+		good-practice.bibb.de
+		ldbb.bibb.de
+		www.ldbb.bibb.de
+		na.bibb.de
+		www.na.bibb.de
+-->
+<ruleset name="BIBB.de">
+	<!-- Federal Institute for Vocational Education and Training -->
```
Comment (YegorIevlev) [inline src/chrome/content/rules/BIBB.de.xml]: Move this to the beginning of the first comment.

File: src/chrome/content/rules/BIBB.de.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	Federal Institute for Vocational Education and Training
+
+	Invalid certificate:
+		europass.bibb.de
+		exanda.bibb.de
+		good-practice.bibb.de
+		ldbb.bibb.de
+		www.ldbb.bibb.de
+		na.bibb.de
+		www.na.bibb.de
```
Comment (YegorIevlev) [inline src/chrome/content/rules/BIBB.de.xml]: Add returned certificates' Common Names.

Comment (Jochen-A-Fuerbacher) [reply src/chrome/content/rules/BIBB.de.xml]: I've never seen any other ruleset where the certificates' common names are provided in the comment section. What's the usecase for this?

Comment (YegorIevlev) [reply src/chrome/content/rules/BIBB.de.xml]: Most likely for cases when domain is pointed to a CDN bucket using CNAME. See this ruleset for an example: https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Exmasters.com.xml. There's currently no rule that requires adding CNs to the comment, so I will approve this PR anyway.

**Pull request merged**